### PR TITLE
Add Time Axis only when time related chart is opened

### PR DESCRIPTION
**Pull request by muddana-satish**
Signed-off-by: muddana-satish <satish.muddana@ericsson.com>

### CODE REVIEW

**Comment by muddana-satish** [comment]
Kapture 2021-07-01 at 11 23 04

File: packages/react-components/src/components/trace-context-component.tsx [hunk 1]
```diff
@@ -283,10 +285,12 @@ export class TraceContextComponent extends React.Component<TraceContextProps, Tr
                     }
                 })}
             </ResponsiveGridLayout>
-            <div style={{ marginLeft: this.state.style.width - this.state.style.chartWidth }}>
-                <TimeNavigatorComponent unitController={this.unitController} style={this.state.style}
-                    addWidgetResizeHandler={this.addWidgetResizeHandler} removeWidgetResizeHandler={this.removeWidgetResizeHandler}/>
-            </div>
+            {(outputs.length > 1 || outputs[0].type !== 'TABLE') &&
```
Comment (MatthewKhouzam) [inline packages/react-components/src/components/trace-context-component.tsx:288]: Future flame charts should be excluded.

Comment (MatthewKhouzam) [reply packages/react-components/src/components/trace-context-component.tsx:288]: I could approve, just add a comment or something.

**Review by MatthewKhouzam — APPROVED** [review]

**Review by bhufmann — APPROVED** [review]
LGTM

**Pull request merged**